src/arfs/folder_hierarchy.test.ts

```
@@ -165,6 +165,101 @@ describe('FolderHierarchy class', () => {
+
+			// Sanity: all folders present and the true drive root is detected.
+			expect(hierarchy.allFolderIDs().length).to.equal(TOTAL_FOLDERS);
+			expect(`${hierarchy.rootNode.folderId}`).to.equal(`${rootId}`);
+
+			// O(folders^2) regression guard: linear node setup finishes in milliseconds, while a
+			// quadratic regression at 10k folders takes multiple seconds. The bound is deliberately
+			// generous so the guard stays non-flaky on slow CI yet still trips on a re-regression.
+			expect(
+				elapsedMs,
+				`newFromEntities took ${elapsedMs}ms for ${TOTAL_FOLDERS} folders`
+			).to.be.lessThan(500);
```

ESLint

[error] Replace `⏎↹↹↹↹elapsedMs,⏎↹↹↹↹`newFromEntities·took·${elapsedMs}ms·for·${TOTAL_FOLDERS}·folders`⏎↹↹↹` with `elapsedMs,·`newFromEntities·took·${elapsedMs}ms·for·${TOTAL_FOLDERS}·folders``

(prettier/prettier)